src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html

```diff
@@ -1,4 +1,11 @@
 <div class="main">
+  <mat-toolbar class="mobile-toolbar">
+    <button mat-icon-button class="mobile-menu" aria-label="Sidebar menu" (click)="toggleSideBar()">
+      <mat-icon class="mobile-icon">{{ !drawer.opened ? 'menu' : 'close' }}</mat-icon>
+    </button>
+    <span class="example-spacer"></span>
```

Comment:
Do you need it? If yes, I believe the class should be renamed not to be `example-...`